tf-training-tfjob-py2/charts/values.yaml

```diff
@@ -26,10 +11,39 @@ resources:
     cpu: 19
     memory: 93604421089
 
+# Shorthands for setting both limits and requests of cpu/memory
 cpu: null
 memory: null
 
+env:
+  - name: OMP_NUM_THREADS
```

Comment:
tf-training-tfjob is not a multinode training - so it doesn't use OpenMPI - thus those variables are superfluous here.